test/libsolidity/MemoryGuardTest.cpp

```diff
@@ -60,8 +60,10 @@ TestCase::TestResult MemoryGuardTest::run(std::ostream& _stream, std::string con
 	for (std::string contractName: compiler().contractNames())
 	{
 		ErrorList errors;
+		std::optional<std::string> const& ir = compiler().yulIR(contractName);
+		solAssert(ir);
 		auto [object, analysisInfo] = yul::test::parse(
-			compiler().yulIR(contractName),
+			*ir,
```

Comment:
One of these days I need to finally write a helper that works like Rust's `unwrap()` and asserts before dereferencing things:
```c++
			unwrap(ir),
```

Reply:
Yeah, seems like it's a frequent operation in the codebase!